odoo/addons/base/tests/test_float.py

```diff
@@ -82,6 +82,30 @@ def try_round(amount, expected, digits=3, method='HALF-UP'):
         try_round(457.4554, '457.455')
         try_round(-457.4554, '-457.455')
 
+        # Try some rounding value with rounding method HALF-DOWN instead of HALF-UP
+        try_round(2.6745, '2.674', method='HALF-DOWN')
+        try_round(-2.6745, '-2.674', method='HALF-DOWN')
+        try_round(2.6744, '2.674', method='HALF-DOWN')
+        try_round(-2.6744, '-2.674', method='HALF-DOWN')
+        try_round(0.0004, '0.000', method='HALF-DOWN')
+        try_round(-0.0004, '-0.000', method='HALF-DOWN')
+        try_round(357.4555, '357.455', method='HALF-DOWN')
+        try_round(-357.4555, '-357.455', method='HALF-DOWN')
+        try_round(457.4554, '457.455', method='HALF-DOWN')
+        try_round(-457.4554, '-457.455', method='HALF-DOWN')
+
+        # Try some rounding value with rounding method HALF-EVEN instead of HALF-UP
+        try_round(2.6745, '2.674', method='HALF-EVEN')
+        try_round(-2.6745, '-2.674', method='HALF-EVEN')
+        try_round(2.6744, '2.674', method='HALF-EVEN')
+        try_round(-2.6744, '-2.674', method='HALF-EVEN')
+        try_round(0.0004, '0.000', method='HALF-EVEN')
+        try_round(-0.0004, '-0.000', method='HALF-EVEN')
+        try_round(357.4555, '357.455', method='HALF-EVEN')
+        try_round(-357.4555, '-357.455', method='HALF-EVEN')
+        try_round(457.4554, '457.455', method='HALF-EVEN')
+        try_round(-457.4554, '-457.455', method='HALF-EVEN')
```

Comment:
You should have some test cases showing that `HALF-EVEN` is different than `HALF-DOWN` (and equal in that case to `HALF-UP`)